ui/packages/app/web/src/components/ui/UserPreferences/UserPreferenceItem.tsx

```diff
@@ -0,0 +1,82 @@
+// Copyright 2022 The Parca Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import cx from 'classnames';
+import useUserPreference, {UserPreferenceDetails} from '@parca/functions/useUserPreference';
+
+interface GenericInputProps {
+  id: string;
+  type: string;
+  value: any;
+  onChange: (value: any) => void;
+  disabled?: boolean;
+}
+
+const GenericInput = ({id, type, value, onChange, disabled = false}: GenericInputProps) => {
+  switch (type) {
+    case 'boolean':
+      return (
+        <input
+          className="bg-inherit border disabled:cursor-not-allowed"
+          id={id}
+          type="checkbox"
+          checked={value}
+          onChange={e => onChange(e.target.checked)}
+          disabled={disabled}
+        />
+      );
+    case 'number':
```

Comment:
interesting idea, but I think this would break shareability, if you have a different configuration from me and you share a link with me I will see different data, if we expose this value (which I don't think we should today), then we should make it part of exploring the data (and therefore part of the URL)

Reply:
Yeah ok, and we can make this a value added to the URL whenever the user is in the ProfileExplorer and the query formed with the value from the URL taking higher precedence over the value from the preference so that the shareability is maintained. 

But I would prefer moving this into a separate issue and implement it once #2115 is merged. As it has some nice utilities for dealing with the URL params state. Any concerns?

Reply:
I would suggest then let's just use a hard-coded value for a start. (I'm not even sure it ever doesn't need to be)

Reply:
I tried to see if I can implement the URL state without that PR, but it looks like a lot of duplicate work between these two. So, I'll hold it off.

So, if I understand it correctly, meanwhile, you would like this option removed from the user preference and work with just the hardcoded `0.005`cut-off value?

Reply:
This is done.